src/app.config.js

```diff
@@ -43,4 +43,12 @@ config["slackInvitationLink"] = "https://bit.ly/in-core";
 config["slackWorkspaceLink"] = "https://in-core.slack.com";
 config["githubRelease"] = "https://github.com/IN-CORE/IN-CORE/releases/tag/";
 
-export default config;
+// Keycloak configuration
+config["keycloakConfig"] = {
+	// url: "http://localhost:8080/auth/", // for testing locally
+	url: `${hostname}/auth/`,
+	realm: "In-core",
+	clientId: "react-auth"
```

Comment:
We have a config variable for that already. 
See line 7: `config["client_id"] = "react-auth";`